src/components/AllBeers.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { Link } from 'react-router-dom';
+
+import Header from './Header';
+
+export class AllBeers extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      beers: [],
+      filtered: [],
+      search: '',
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get('https://ih-beers-api2.herokuapp.com/beers')
+      .then((response) => {
+        this.setState({ beers: response.data, filtered: response.data });
+      })
+      .catch((err) => console.log('Error trying to get the beer list', err));
+  }
+
+  handleSearch = (event) => {
+    let newList = [];
+
+    if (event.target.value !== '') {
```

Comment:
```
const emptyString = ""
const someStringValue = "alba gustems"
```
`someStringValue.includes(emptyString)` --> siempre devuelve `true` por lo cual no necesitas este if else, si el filtro es vacio devuelve el array entero

`const filteredBeers = this.state.beers.filter(beer => beer.name.toLowerCase().includes(filter))`
`this.setState(state => ({...state, filtered: filteredBeers, [target.name]: target.value})`